tsdef/twilio-video-tests.ts

```diff
@@ -103,3 +103,38 @@ function localDataTrackPublication(publication: Video.LocalDataTrackPublication)
   const publicationInfo = { kind, track };
   return publicationInfo;
 }
+
+function RemoteParticipant(RemoteParticipant: Video.RemoteParticipant) {
```

Comment:
I'm curious about these tests - what kind of "code coverage" are we aiming for here?

The `RemoteParticipant` class for instance - it has four properties and 14 events (in addition to those it inherits from the `Participant` class. This file is testing just two of the events. 

I'm not sure if it's worth the effort to go for 100% "code coverage" here (where each property, method, and event of every class is tested), but it could be. Whether or not we shoot for 100% "coverage", I do think these tests could be more robust.

Reply:
Thanks for looking @timmydoza! Michael pointed this out as well, currently working on adding more coverage. When I added these tests, I also wasn't sure as well. But I think it is definitely worth adding more now!